docusaurus/docs/Android/02-client/06-guides/02-sending-custom-attachments.mdx

```diff
@@ -25,13 +29,39 @@ ChatClient.instance().sendMessage(channelType = "messaging", channelId = "genera
     }
 }
 ```
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+Attachment attachment = new Attachment();
+Message message = new Message();
+message.setCid("messaging:general");
+message.setText("Look at this attachment!");
+message.setAttachments(Collections.singletonList(attachment));
+
+ChatClient.instance().sendMessage("messaging", "general", message)
+        .enqueue((result) -> {
+                    if (result.isSuccess()) {
+                        // Use result.data()
+                    } else {
+                        // Handle result.error()
+                    }
+                }
+        );
```

Comment:
```suggestion
ChatClient.instance().sendMessage("messaging", "general", message)
        .enqueue((result) -> {
            if (result.isSuccess()) {
                // Use result.data()
            } else {
                // Handle result.error()
            }
        });
```

Reply:
It's a default formatting. I'm leaving it as is